iree/base/config.h

```diff
@@ -166,4 +166,11 @@ typedef IREE_DEVICE_SIZE_T iree_device_size_t;
 #define IREE_VM_EXT_F64_ENABLE 0
 #endif  // !IREE_VM_EXT_F64_ENABLE
 
+#if !defined(IREE_TIME_NOW_FN)
```

Comment:
(paired with the comment below where used) we don't want to provide a default here as if used this would disable timing on all platforms - you'd want to provide this in your own IREE_USER_CONFIG_H or in your toolchain cmake (`-DIREE_TIME_NOW_FN=return(0);` etc)

Reply:
Ahh.. I see. Thanks for the tip!